mriqc/cli/run.py

```diff
@@ -30,14 +29,30 @@ def main():
     import os
     import sys
     from tempfile import mktemp
-    from multiprocessing import Manager, Process
-
-    from ..utils.bids import write_bidsignore, write_derivative_description
-    from .parser import parse_args
+    from mriqc import config, messages
+    from mriqc.cli.parser import parse_args
 
     # Run parser
     parse_args()
 
+    _plugin = config.nipype.get_plugin()
+    if config.nipype.plugin in ("MultiProc", "LegacyMultiProc"):
+        from importlib import import_module
+        from multiprocessing import set_start_method
+        from contextlib import suppress
+
+        with suppress(RuntimeError):
+            set_start_method("forkserver")
+
+        Plugin = getattr(
+            import_module(f"nipype.pipeline.plugins.{config.nipype.plugin.lower()}"),
+            f"{config.nipype.plugin}Plugin",
+        )
+        _plugin = {
+            "plugin": Plugin(plugin_args=config.nipype.plugin_args),
+        }
+        gc.collect()
```

Comment:
I don't know what L95 could be possibly be collecting so... this is to say, I'm happy either way and maybe there are some unused variables in the plugin implementation that this would catch. Honestly, IDK. Happy to remove if it feels superfluous.